src/ploneintranet/activitystream/browser/navigation.py

```diff
@@ -39,17 +39,20 @@ def items(self):
                               title=m_context.Title() + ' updates',
                               state='localstream')])
 
-        base = self.portal_url() + '/'
-        menu.extend([dict(url=base + '@@stream',
-                          title='Explore',
-                          state='explore')])
-        if PLONEINTRANET.network:
-            menu.extend([dict(url=base + '@@stream/network',
-                              title='My network',
-                              state='stream'),
-                         dict(url=base + '@@author',
-                              title='My profile',
-                              state='profile')])
+
+#        TODO: these lines are commented until the corresponding
+#        functionalities  will be complete
```

Comment:
No these lines should be removed completely. The dash navbar should not be dynamic.